bin/magpie-launch-phoenix.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+#############################################################################
+#  Copyright (C) 2013-2015 Lawrence Livermore National Security, LLC.
+#  Produced at Lawrence Livermore National Laboratory (cf, DISCLAIMER).
+#  Written by Albert Chu <chu11@llnl.gov>
+#  LLNL-CODE-644248
+#  
+#  This file is part of Magpie, scripts for running Hadoop on
+#  traditional HPC systems.  For details, see <URL>.
+#  
+#  Magpie is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU General Public License as published by
+#  the Free Software Foundation; either version 2 of the License, or
+#  (at your option) any later version.
+#  
+#  Magpie is distributed in the hope that it will be useful, but
+#  WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#  General Public License for more details.
+#  
+#  You should have received a copy of the GNU General Public License
+#  along with Magpie.  If not, see <http://www.gnu.org/licenses/>.
+#############################################################################
+
+# This script launches Phoenix for the user
+
+# First argument is path to HBase config
+
+# Second argument is path to Phoenix
+
+# Third argument is start or stop
+
+if [ "$1X" == "X" ]
+then
+    echo "User must specify path to Phoenix config as first argument"
+    exit 1
+fi
+
+if [ ! -d $1 ]
+then
+    echo "Phoenix config path is not a directory"
+fi
+
+if [ "$2X" == "X" ]
+then
+    echo "User must specify path to Hbase config as second argument"
+    exit 1
```

Comment:
likewise inconsistent to comments above, looking at the code, i think the comments above are wrong??